src/pages/MediaPage.php

```diff
@@ -413,7 +413,7 @@ public function Link($action = null) {
 			$parent->Link(),
 			"{$date}{$this->URLSegment}/"
 		);
-		if($action) {
+		if(is_string($action) && !empty($action)) {
```

Comment:
Just personal preference, but could this be `if($action && is_string($action))`? This will also need the same change for `AbsoluteLink` further below to ensure that doesn't hit a similar issue at some point. Nice pick up :smile:

Reply:
Ah yep, done.